src/lib/backend.ai-client-esm.ts

```diff
@@ -734,6 +734,9 @@ class Client {
       this._features['max_network_count'] = true;
       this._features['replicas'] = true;
     }
+    if (this.isManagerVersionCompatibleWith(['25.1.0', '24.09.5', '24.03.11'])) {
```

Comment:
```suggestion
    if (this.isManagerVersionCompatibleWith(['25.1.0', '24.09.6', '24.03.12'])) {
```
This is the correct upcoming release versions